includes/wikia/Wikia.php

```diff
@@ -334,27 +334,24 @@ static public function fixDomainName( $name, $language = false, $type = false )
 		$name = strtolower( $name );
 
 		$parts = explode(".", trim($name));
-		Wikia::log( __METHOD__, "info", "$name $language $type" );
-		if( is_array( $parts ) ) {
-			if( count( $parts ) <= 2 ) {
-				$allowLang = true;
-				switch( $type ) {
-					case "answers":
-						$domains = self::getAnswersDomains();
-						if ( $language && isset($domains[$language]) && !empty($domains[$language]) ) {
-							$name =  sprintf("%s.%s.%s", $name, $domains[$language], $wgWikiaBaseDomain);
-							$allowLang = false;
-						} else {
-							$name =  sprintf("%s.%s.%s", $name, $domains["default"], $wgWikiaBaseDomain);
-						}
-						break;
-
-					default:
-						$name = sprintf("%s.%s", $name, $wgWikiaBaseDomain);
-				}
-				if ( $language && $language != "en" && $allowLang ) {
-					$name = $language.".".$name;
-				}
+		if( is_array( $parts ) && count( $parts ) <= 2 ) {
+			$allowLang = true;
+			switch( $type ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)